### ISSUE TITLE
EnumType not supported

### ISSUE THREAD

**Issue by fabienfleureau** [aileftech/spring-boot-database-admin#22]
**Describe the bug**
There is an error for entity that contains field of type enum.

**Is the bug at startup before you perform any action?**
Not applicable

**@Entity code**
```
    @Column(name = "status", nullable = false)
    @Enumerated(EnumType.STRING)
    var status: StatusEnum
```

In case of EnumType.STRING -> print the string itself

**Comment by aileftech** [comment]
Pushed the implementation on the `dev` branch. Both ORDINAL and STRING types are supported.

**Comment by aileftech** [comment]
Released in `0.1.8`.
